docs/install.md

```diff
@@ -10,10 +10,18 @@ Get started with Draft in three easy steps:
 
 In order to get started, you will need to fetch the following:
 
-- [the latest release of minikube](https://github.com/kubernetes/minikube/releases)
-- [the latest release of kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-- [the latest release of Helm](https://github.com/kubernetes/helm/releases)
-- [the latest release of Draft](https://github.com/Azure/draft/releases)
+- the latest release of minikube
+- the latest release of kubectl
+- the latest release of Helm
+
+All of the dependencies can be installed by the following:
+
+```
+$ brew cask install minikube
+$ brew install kubernetes-helm kubectl
```

Comment:
[So it does](https://github.com/caskroom/homebrew-cask/blob/23872ccc5e935d61874d5211635aa152db69ed9f/Casks/minikube.rb#L12)! I guess this step can be removed.